pyvista/examples/downloads.py

```diff
@@ -1653,11 +1653,80 @@ def download_frog(load=True):  # pragma: no cover
     this dataset.
 
     """
-    # TODO: there are other files with this
     download_file('froggy/frog.zraw')
     return _download_and_read('froggy/frog.mhd', load=load)
 
 
+def download_frog_tissue(load=True):  # pragma: no cover
+    """Download frog tissue dataset.
+
+    This dataset contains tissue segmentation labels for the frog dataset
+    (see :ref:`download_frog`).
+
+    Parameters
+    ----------
+    load : bool, default: True
+        Load the dataset after downloading it when ``True``.  Set this
+        to ``False`` and only the filename will be returned.
+
+    Returns
+    -------
+    pyvista.ImageData | str
+        DataSet or filename depending on ``load``.
+
+    Examples
+    --------
+    # Load data
```

Comment:
We don't need to comment out the comments :)
```suggestion
    Load data

```